py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -986,6 +974,25 @@ def aten_ops_amax(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.amin.default)
```

Comment:
This converter (as well as `amax` above) seem to require `ITensor` for `args[0]`, so the `@enforce_tensor_types` decorator can be added.